js/poloniex.js

```diff
@@ -465,17 +470,17 @@ module.exports = class poloniex extends Exchange {
             'ask': this.safeNumber (ticker, 'lowestAsk'),
             'askVolume': undefined,
             'vwap': undefined,
-            'open': open,
+            'open': undefined,
             'close': last,
             'last': last,
             'previousClose': undefined,
-            'change': change,
-            'percentage': relativeChange * 100,
-            'average': average,
+            'change': undefined,
+            'percentage': percentage,
+            'average': undefined,
             'baseVolume': this.safeNumber (ticker, 'quoteVolume'),
             'quoteVolume': this.safeNumber (ticker, 'baseVolume'),
             'info': ticker,
-        };
+        });
```

Comment:
Missed a `market` argument to `safeTicker` here.

Reply:
@kroitor sorry and thanks for fixing it 🙂